src/resources/help/en_US/faq.html

```diff
@@ -452,12 +440,12 @@ <h2><a name="20"></a>20. Where can I get the Plugin Developer's Guide referenced
 <p>There are two popular methods for starting a plugin. You can copy the files from another plugin, rename, and gradually adapt them to your needs or you can begin from scratch. If you begin from scratch, you will want to run VisIt's XMLEdit program, which loads, edits, and saves XML files for VisIt plugins. XMLEdit is used to set up the required attributes of a plugin such as its type and name. XMLEdit is also used to set up more advanced attributes such as the file extensions associated with a database plugin or the plot attributes required for a plot plugin. It can be helpful to look at the XML files from existing VisIt plugins to get an idea of how the XML files are used. </p>
 <p>Once you have created an XML file for your plugin, run <em>xml2plugin -clobber myxmlfile.xml</em> to generate the plugin skeleton, any automatically generated code, and Makefile or project files needed to build your plugin. Once you've generated your plugin for the first time, check the directory in which you're keeping your plugin sources and XML file. There should be several new files. Most are boilerplate but pay special attention to the files that begin with &quot;avt&quot; since those are the files to which you generally must add your plugin code.</p>
 <h2><a name="22"></a>22. How often is a new version of VisIt released?</h2>
-<p>The VisIt development team usually releases a new version of VisIt every 3 months. Each VisIt release contains on average 60 enhancements or bug-fixes, improving the functionality and the quality of the tool. Check the VisIt Web site often for updates. You can also join the VisIt User's mailing list if you want to know when VisIt is updated. You can also subscribe to the VisIt project on <a href="http://freshmeat.net/projects/visit/" title="Freshmeat" target="_blank">Freshmeat.net</a> if you want to be informed when a new version of VisIt becomes available. Finally, you can use the &quot;VisIt update&quot; feature in VisIt's help menu to automatically determine if an update for your platform is available. &quot;VisIt update&quot; can in many cases even download and install the new version of VisIt. </p>
+<p>The VisIt development team usually releases a new version of VisIt every 3 months. Each VisIt release contains on average 60 enhancements or bug-fixes, improving the functionality and the quality of the tool. Check the <a href="https://visit.llnl.gov">VisIt Web site</a> often for updates. Finally, you can use the &quot;VisIt update&quot; feature in VisIt's help menu to automatically determine if an update for your platform is available. &quot;VisIt update&quot; can in many cases even download and install the new version of VisIt. </p>
```

Comment:
The FAQ should be removed. I didn't realize this was here. At one point I updated and greatly reduced the FAQ on the LLNL website and now it is completely gone from the GitHub website.

Reply:
Done